e2e/pages/adf/demo-shell/custom-sources.page.ts

```diff
@@ -56,18 +51,19 @@ export class CustomSources {
         await this.waitForToolbarToBeVisible();
     }
 
-    async clickOnSourceType(): Promise<void> {
-        await BrowserActions.click(this.sourceTypeDropdown);
-    }
-
     async selectMySitesSourceType(): Promise<void> {
-        await this.clickOnSourceType();
-        await BrowserActions.click(this.getSourceType(source.mySites));
+        await this.selectModeDropdown.clickDropdown();
+        await this.selectModeDropdown.selectOption(source.mySites);
```

Comment:
there is a method called selectDropdownWithOption that is wrapping this 2 methods. Can we use that one?